R/utilities-quantity.R

```diff
@@ -113,19 +125,31 @@ setQuantityValues <- function(quantities, values) {
 #'
 #' setParameterValuesByPath(list("Organism|Liver|Volume", "Organism|Liver|A"), c(2, 3), sim)
 #' @export
-setQuantityValuesByPath <- function(quantityPaths, values, simulation, stopIfNotFound = TRUE) {
+setQuantityValuesByPath <- function(quantityPaths, values, simulation, stopIfNotFound = TRUE, units = NULL) {

```

Comment:
looking good. stopInfNotFound is in general OUR LAST parameter. Here we are changing the signatures `logic` .
shouldn't we put `units` before `stopIfNotFound`  If people have used this flag, one can argue that they would have named it explicitly to set it to FALSE

Reply:
I couldn't decide what is better - keep the logic by putting `stopIfNotFound` at the end, or maintain compatibility with code that uses the function without specifying argument names. But I think the method has not been used extensively so far, so let's keep the logic. I will update.